daemon/networkdriver/bridge/driver.go

```diff
@@ -113,6 +114,13 @@ func InitDriver(config *Config) error {
 		addrsv6    []net.Addr
 		bridgeIPv6 = "fe80::1/64"
 	)
+
+	// try to modprobe bridge first
+	// see gh#12177
+	if out, err := exec.Command("modprobe", "bridge", "nf_nat").Output(); err != nil {
+		logrus.Errorf("Running modprobe bridge nf_nat failed with message: %s, error: %v", out, err)
```

Comment:
I was testing error conditions and ran into this oddity on my machine (Ubunut 14.10).  The following prints ok

    sudo modprobe bridge asdf && echo ok

Maybe you should do two separate modprobes?